### ISSUE TITLE
RAF-DB dataset

### ISSUE THREAD

**Issue by XFFFCCCC** [ArnabKumarRoy02/ResEmoteNet#22]
The RAF-DB dataset introduced in the paper only has a training set and a test set, but the code has a training set, a test set, and a validation set. What is the validation set? If possible, can you share your processed RAF-DB dataset? Thank you

**Comment by ArnabKumarRoy02** [comment]
As there are no validation set in the original RAF-DB dataset, we had to create the validation set using augmentation from the training set. [Here's](https://drive.google.com/file/d/1JmHQllsQqvHnexh63zi0pJWzVpfvV7Dg/view?usp=share_link) the copy of the dataset we used for this repository.

**Comment by XFFFCCCC** [comment]
> As there are no validation set in the original RAF-DB dataset, we had to create the validation set using augmentation from the training set. [Here's](https://drive.google.com/file/d/1JmHQllsQqvHnexh63zi0pJWzVpfvV7Dg/view?usp=share_link) the copy of the dataset we used for this repository.

Thank you
